packages/api/db/migration/20220725200254_remove_webhook_address_from_integrations.js

```diff
@@ -0,0 +1,11 @@
+exports.up = async function (knex) {
+  return await knex.schema.alterTable('farm_external_integration', (table) => {
+    table.dropColumn('webhook_address');
+  });
+};
+
+exports.down = async function (knex) {
+  return await knex.schema.alterTable('farm_external_integration', (table) => {
+    table.integer('webhook_address');
```

Comment:
To make sure the migration can be rolled back, we should be recreating the column to have the same type as before
```suggestion
    table.string('webhook_address');
```